src/containers/pages/InterventionPlan/UpdatePlan/tests/index.test.tsx

```diff
@@ -63,8 +63,7 @@ describe('components/InterventionPlan/UpdatePlan', () => {
     );
     expect(toJson(wrapper.find('Breadcrumb'))).toMatchSnapshot('Breadcrumb');
     expect(toJson(wrapper.find('h3.page-title'))).toMatchSnapshot('Page title');
-    expect(wrapper.find('PlanForm').props()).toEqual({
-      ...updatePlanFormProps,
+    expect(wrapper.find('PlanForm').props()).toMatchSnapshot({
```

Comment:
also moved to snapshot

Reply:
Similar to above, changed this back to using a much smaller fixture (turns out we could import it from elsewhere)